PWGHF/Utils/UtilsBfieldCCDB.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file UtilsBfieldCCDB.h
```

Comment:
```suggestion
/// \file utilsBfieldCCDB.h
```
See the [naming conventions](https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=File_Names#File_Names)

Reply:
I changed the name, but then this does not match with `UtilsDebugLcK0Sp.h`